pkgs/applications/science/math/mathematica/default.nix

```diff
@@ -92,6 +92,10 @@ stdenv.mkDerivation rec {
     sed -i -e 's/^PATH=/# PATH=/' MathInstaller
     sed -i -e 's/\/bin\/bash/\/bin\/sh/' MathInstaller
 
+    # Install the desktop items
+    sed -i -e 's/Executables\/Mathematica/..\/..\/bin\/mathematica/' MathInstaller
+    export XDG_DATA_HOME="$out/share"
```

Comment:
Why is this required?

Reply:
Never mind i now know but can it be done in the install call like execdir is?

Reply:
There doesn't appear to be any option in `MathInstaller` for configuring where these files go. `MathInstaller` calls the `xdg-desktop-menu` script (and the like) to install these files.